data/tlg5019/tlg001/tlg5019.tlg001.1st1K-grc1.xml

```diff
@@ -0,0 +1,2208 @@
+<titleStmt>
+<title xml:lang="lat">Scholia in Dionysium Periegetam</title>
+<author>Anonymous</author>
+<editor xml:lang="lat">Carolus Müllerus</editor>
+<funder>Harvard Library Arcadia Fund</funder>
+<principal>Gregory Crane</principal>
+<respStmt>
+<persName xml:id="DDD">Digital Divide Data</persName>
+<resp>Corrected and encoded the text</resp>
+</respStmt>
+<respStmt>
+<persName>Gregory Crane</persName>
+<resp>Editor-in-Chief, Perseus Digital Library</resp>
+</respStmt>
+<respStmt>
+<persName>Matt Munson</persName>
+<resp>Project Manager (University of Leipzig), 2016 - present</resp>
+</respStmt>
+<respStmt>
+<persName>Annette Gessner</persName>
+<resp>Project Assistant (University of Leipzig) 2015 - present</resp>
+</respStmt>
+<respStmt>
+<persName>Thibault Clérice</persName>
+<resp>Lead Developer (University of Leipzig) 2015 - 2017</resp>
+</respStmt>
+<respStmt>
+<persName>Bruce Robertson</persName>
+<resp>Technical Advisor</resp>
+</respStmt>
+</titleStmt>
+<publicationStmt>
+<authority>Harvard College Library</authority>
+
+<idno type="filename">tlg5019.tlg001.1st1K-grc1.xml</idno>
+
+<availability>
+<p>Available under a Creative Commons Attribution-ShareAlike 4.0 International License</p>
+</availability>
+<date>2017</date>
+<publisher>Harvard College Library</publisher>
+<pubPlace>United States</pubPlace>
+</publicationStmt>
+<sourceDesc>
+<listBibl>
+<biblStruct>
+<monogr>
+
+<title xml:lang="lat">Geographi Graeci Minores</title>
+<editor>
+<persName>
+<name xml:lang="lat">Carolus Müllerus</name>
+<addName xml:lang="deu">Karl Müller</addName>
+</persName>
+</editor>
+<author ref="urn:cts:greekLit:tlg5019">Scholia in Dionysium Periegetam</author>
+<imprint>
+<publisher>Ambroise-Firmin Didot</publisher>
+<pubPlace>Paris</pubPlace>
+<date>1861</date>
+</imprint>
+
+<biblScope unit="volume">2</biblScope>
+
+</monogr>
+<ref target="https://archive.org/details/bub_gb_63VfAAAAMAAJ">The Internet Archive</ref>
+</biblStruct>
+</listBibl>
+</sourceDesc>
+</fileDesc>
+
+
+<encodingDesc>
+
+<p>Text encoded in accordance with the latest EpiDoc standards</p>
+<p>The following text is encoded in accordance with EpiDoc standards and with the CTS/CITE
+Architecture.</p>
+
+
+
+<refsDecl n="CTS">
+
+<cRefPattern n="section" matchPattern="(\w+).(\w+)"
+replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1']/tei:div[@n='$2'])"/>
+<cRefPattern n="book" matchPattern="(\w+)"
+replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])"/>
+
+
+
+
+</refsDecl>
+
+</encodingDesc>
+<profileDesc>
+<langUsage>
+<language ident="grc">Greek</language>
+<language ident="lat">Latin</language>
+<language ident="deu">German</language>
+</langUsage>
+</profileDesc>
+<revisionDesc>
+<change who="Chiara Palladino">created the file</change>
+</revisionDesc>
+</teiHeader>
+<text>
+<body>
+<div type="edition" xml:lang="grc" n="urn:cts:greekLit:tlg5019.tlg001.1st1K-grc1">
+
+
+
+
+<div type="textpart" subtype="book" n="scholia">
```

Comment:
Since there is only a single "book" that encompasses the whole text, I would tend to take it out. However, if the "book" is necessary to preserve an accepted citation scheme, then keep it. I have the feeling here, however, that it is not necessary. Since this work is all Scholia, I wouldn't think there is a need to have an all-encompassing "book" element called "scholia."